docs/celeborn_ratis_shell.md

```diff
@@ -22,6 +22,8 @@ license: |
 Celeborn uses Ratis to implement the HA function of the master, Celeborn directly introduces ratis-shell package into the project
 then it's convenient for Celeborn Admin to operate the master ratis service.
 
+Since celeborn-0.6.0, the ratis [RESTful API](webapi.md) is supported, which is more convenient to operate the ratis service, see details in the swagger: `http://<CELEBORN_HOST>:<CELEBORN_PORT>/swagger/#/Ratis`.
```

Comment:
I think it is enough for the docs.

<img width="1317" alt="image" src="https://github.com/user-attachments/assets/db865860-407e-49f8-86f2-c8835ca49268">